components/ide/jetbrains/gateway-plugin/build.gradle.kts

```diff
@@ -155,7 +155,7 @@ tasks {
         var pluginChannel: String? = System.getenv("JB_GATEWAY_GITPOD_PLUGIN_CHANNEL")
         if (pluginChannel.isNullOrBlank()) {
             pluginChannel = if (pluginVersion.contains("-main.")) {
-                "Nightly"
+                "Stable"
```

Comment:
After re-checking the docs, I think the Stable channel is actually called "default".
So for publishing to stable we would use `"default"` instead of `"Stable"` or don't call `channels.set(listOf(pluginChannel))` down there (cause if there are no custom channels defined, it will be published to default). [[1](https://plugins.jetbrains.com/docs/intellij/publishing-plugin.html#specifying-a-release-channel)][[2](https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-publishplugin)]

Not sure if leaving "Stable" will have the same effect as "default". But we can try and see.

Reply:
Note that we can access the custom plugin channels "dev" and "nightly" via urls:
- https://plugins.jetbrains.com/plugins/list?channel=dev&pluginId=18438
- https://plugins.jetbrains.com/plugins/list?channel=nightly&pluginId=18438

But if we try to access the following, it returns nothing:
- https://plugins.jetbrains.com/plugins/list?channel=default&pluginId=18438
- https://plugins.jetbrains.com/plugins/list?channel=stable&pluginId=18438

To have the default channel list, we need to use https://plugins.jetbrains.com/plugins/list?pluginId=18438 (or leave the channel empty: https://plugins.jetbrains.com/plugins/list?channel=&pluginId=18438)

Reply:
I found this example from JetBrains where they use `stable` lowercase [[1]](https://github.com/JetBrains/projector-server/blob/master/projector-plugin/build.gradle.kts#L45-L48)

Reply:
From this endpoint we can see the channels available to the Gitpod plugin: https://plugins.jetbrains.com/api/plugins/18438/channels

I think "stable" -> "Stable" -> "default" or even removing the `channels.set` will be the same 🤔

Reply:
By looking again, at this endpoint: https://plugins.jetbrains.com/api/plugins/18438/updates?channel=&size=100

We can see that stable releases have empty channel, if we search for Nightly or Dev releases they are lowercase and present:

- https://plugins.jetbrains.com/api/plugins/18438/updates?channel=Dev&size=100
- https://plugins.jetbrains.com/api/plugins/18438/updates?channel=Nightly&size=100

So, I agree @felladrin, let's remove it

Reply:
Go for it. We can promptly change it in case `"stable"` doesn't work.

Reply:
let's try!